test1.js

```diff
@@ -3,3 +3,4 @@ var dev = "DEV";
 var test = "TEST";
 var qa = "QA";
 var prod = "PROD";
+var demo = "DEMO";
```

Comment:
## Need to consider the TRAINING environment
Reviewer @zeinoun 

First Header | Second Header
------------ | -------------
Content from cell 1 | Content from cell 2
Content in the first column | Content in the second column

Reply:
```suggestion
var demo = "DEMO";
var train = "TRAIN";
```